libbpf-tools/trace_helpers.c

```diff
@@ -159,6 +166,134 @@ const struct ksym *ksyms__get_symbol(const struct ksyms *ksyms,
 	return NULL;
 }
 
+struct parts {
```

Comment:
I think parts is not a great name. Let's spell it out completely: `partitions`

Reply:
Agree, thanks.